taskt/Core/Automation/Commands/Data Commands/LogDataCommand.cs

```diff
@@ -1,45 +1,44 @@
-﻿using System;
+﻿using Serilog;
+using System;
 using System.Collections.Generic;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command logs data to files.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to log custom data to a file for debugging or analytical purposes.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements 'Thread.Sleep' to achieve automation.")]
+    [Group("Data Commands")]
+    [Description("This command logs text data to either an engine file or a custom file.")]
     public class LogDataCommand : ScriptCommand
     {
-
-
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select existing log file or enter a custom name.")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Engine Logs")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate the file name where logs should be appended to")]
-        [Attributes.PropertyAttributes.SampleUsage("Select 'Engine Logs' or specify your own file")]
-        [Attributes.PropertyAttributes.Remarks("Date and Time will be automatically appended to the file name.  Logs are all saved in taskt Root\\Logs folder")]
+        [PropertyDescription("Write Log To")]
+        [InputSpecification("Specify logging option to save logs to Engine Logs or a File.")]
+        [SampleUsage("Engine Logs || Custom File Name || {vFileVariable}")]
+        [Remarks("'Engine Logs' to write log into Engine Logs. Date and Time will be automatically appended to the file name. Logs are all saved in taskt Root\\Logs folder.")]
```

Comment:
Change to "Selecting 'Engine Logs' will result in writing execution logins in the 'Engine Logs'. The current Date and Time will be automatically appended to a local file if a custom file name is provided. Logs are all saved in the TaskT Root Folder in the 'Logs' folder."

Reply:
done